### DISCUSSION TITLE
search and bookmark

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2080] [Q&A]
I have a txt file that when imported for research, IPED already informs how many lines were read and how many it found something. 
For example, a file with 100 lines like this:
hash:62abdbdc465b7b30b088d0107e935c12
hash:8cf77d037271daf4926221ceba741c38
hash:da89d5a2297af19eaa7b4777915f5107
hash:8505f74bbf0256fdff137f11880c695c

Is there any way to create a bookmark with the results of this search, so when I click on that bookmark all the files appear?

**Comment by lfcnassif** [comment]
Yes, it was recently implemented by @gfd2020, documented here:
https://github.com/sepinf-inc/IPED/issues/1768

It will be in the next 4.2.0 version in a few months. If you can't wait, please try the latest snapshot here:
https://github.com/sepinf-inc/IPED/actions/runs/7793122008/artifacts/1222202614

**Reply by paulobreim** [reply]
Thank you,  I will try it.

**Reply by paulobreim** [reply]
Worked very well !
